### PR TITLE
Fix various text wrapping on different screen sizes

### PR DESCRIPTION
**Pull request by kiwansim**
[sc-11800]
[sc-11798]
[sc-12072]

### CODE REVIEW

File: client/src/styles/PortfolioTable.scss [hunk 1]
```diff
@@ -15,6 +15,14 @@ $table-border-dark: 1px solid $dark;
     position: relative;
     overflow: auto;
 
+    th[class^="col-Evictions"] div {
+      min-width: 150px;
+    }
+
+    tr {
+      padding: 0px 5px;
```
Comment (austensen) [inline client/src/styles/PortfolioTable.scss:23]:
```suggestion
      padding: 0rem 0.5rem;
```

**Review by austensen — APPROVED** [review]
This all looks great! just added some px -> rem changes if you want them

File: client/src/styles/AddressToolbar.scss [hunk 2]
```diff
@@ -5,6 +5,12 @@
     display: none;
   }
 
+  .btn-group {
+    flex-wrap: nowrap;
+    min-width: 190px;
+    padding-left: 8px;
```
[the remark this answers is not in the document]
Comment (kiwansim) [reply client/src/styles/AddressToolbar.scss:11]: thank you for adding the rems! do we have a preference for `rem` on our scss files? I've noticed that there's a good mix of both px and rem so I wasn't sure

[the remark this answers is not in the document]
Comment (austensen) [reply client/src/styles/AddressToolbar.scss:11]: Yeah, my understanding is that we should always use `rem` units, and I believe the main reason is that it ensures people can change their browser settings for bigger text, whereas using pixels can block that. https://austingil.com/px-or-rem-in-css
And I think we're good about that in everything but WOW, where it seems like it's slowly transitioning over from pixels with new additions. We should probably just do a standalone PR that makes the change everywhere.

**Pull request merged**